File: professionnel-appointemment/lib/models/review.dart
class Review {
  final int? id;
  final int? professionId;
  final int? clientId;
  final String clientName;
  final double? rating;
  final String comment;

  Review({
    required this.id,
    required this.professionId,
    required this.clientId,
    required this.clientName,
    required this.rating,
    required this.comment,
  });

  factory Review.fromJson(Map<String, dynamic> json) {
    return Review(
      id: json['id'],
      professionId: json['professionId'],
      clientId: json['clientId'],
      clientName: json['clientName'],
      rating: json['rating'],
      comment: json['comment'],
    );
  }
}

File: server/Models/Review/Review.cs
namespace WebApi.Models.Review;

public class ReviewCustom
{

    public int? Id { get; set; }
    public int? ClientId { get; set; }
    public string ClientName { get; set; }

    public int? ProfessionId { get; set; }
   
    public double? Rating { get; set; }

    public String Comment { get; set; }


}